docs/services/budget-manager.md

```diff
@@ -0,0 +1,31 @@
+# Budget Manager
+
+The Budget Manager has experience with university funding policies and the specific requirements of funding agencies who support our hackweeks.
+
+## Goals
+
+* Assisting team members to develop realistic cost estimates for hackweek activities and expenses.
+* Ensuring team activities and expenditures align with the requirements of funding agencies.
+
+## Specific Activities
+
+* notify hackweek organizers of potential funding sources
```

Comment:
For style consistency, should the first words in this bullet list be capitalized to match the `Goals` from above?

Reply:
Note: This is also inconsistent in other role descriptions.